cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -25,6 +25,10 @@ with the name of the JAR file as a header::
 
   X-Archive-Name: <JAR filename>
 
+configuration object as a serialized JSON string as another header (optional)::
```

Comment:
(An optional header can supply a configuration object as a serialized JSON string:)

::

[I think this will work better, as continuing the sentence as we did previous doesn't work to well with the optional component in the middle. Note the :: by itself to trigger the code formatting.]